content/en/security_platform/cloud_workload_security/self_test.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Troubleshooting
+kind: documentation
+description: "Troubleshooting guide for Datadog Cloud Workload Security."
+---
+
+## Self tests
+
+In order to ensure that:
+- the communication between the `security-agent` and the `system-probe` is
+working as expected
+- the Cloud Workload Security probe is able to detect system events
+
+you can manually trigger self tests by running the following command:
+
+| Platform     | Command                                                                             |
+| --------     | -------                                                                             |
+| Docker       | `docker exec -it datadog-agent security-agent runtime self-test`                    |
+| Kubernetes   | `kubectl exec -it <POD_NAME> -c security-agent -- security-agent runtime self-test` |
+| Host         | `sudo /opt/datadog-agent/embedded/bin/security-agent runtime self-test`             |
+
+The self-test procedure creates some temporary files and rules monitoring
+them, and then triggers those rules ensuring that events are correctly propagated.
+
+If everything is working as expected, you should see:
+```
+Runtime self test: OK
+```
+and then see events coming from the `runtime-security-agent` in your logs
+
+{{< img src="security_platform/cws/self_test_logs.png" alt="Self Test events in Logs explorer" style="width:90%;">}}
+
+In the case of an error, you should see:
```

Comment:
```suggestion
If not properly configured, the following error may occur, where `<ERROR_MESSAGE>` is a description of the error encountered during the test.

```

Reply:
Do we have common errors / resolution steps for those errors? Might be helpful to add in a few here with mention of how to resolve. If not, I'd remove this section entirely as it's self-explanatory and could spark more confusion than help.

Reply:
For now we don't. I will remove this section